libs/env/src/env.ts

```diff
@@ -577,7 +577,7 @@ const envDefinitions = {
     desc: "if defined, avoids bypass of the currentDevice in the store.",
   },
   NFT_CURRENCIES: {
-    def: ["avalanche_c_chain", "bsc", "ethereum", "polygon"],
+    def: ["avalanche_c_chain", "bsc", "ethereum", "polygon", "solana"],
```

Comment:
We don't want to activate it right now, because it will be activated on PROD directly if we push that. We are working on feature flagging on our side currently :) 

We need to merge that at the end of @LedgerHQ/live-hub devs 
https://github.com/LedgerHQ/ledger-live/pull/9551